internal/protoc/downloader.go

```diff
@@ -110,15 +110,11 @@ func (d *downloader) cache() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	downloaded, err := d.isDownloaded(basePath)
-	if err != nil {
-		return "", err
-	}
-	if !downloaded {
+	if d.checkDownloaded(basePath) != nil {
```

Comment:
nit: this reads a little funky. Could we do the more common way:
```
if err := d.checkDownload(basePath); err != nil { ... }
```

Reply:
Done